api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirDiagnosticReportServiceImpl.java

```diff
@@ -61,4 +63,35 @@ public DiagnosticReport updateDiagnosticReport(String uuid, DiagnosticReport dia
 		
 		return translator.toFhirResource(dao.saveObsGroup(translator.toOpenmrsType(obsGroup, diagnosticReport)));
 	}
+	
+	@Override
+	protected FhirDao<Obs> getDao() {
+		// TODO Auto-generated method stub
+		return null;
+	}
+	
+	@Override
+	protected OpenmrsFhirTranslator<Obs, DiagnosticReport> getTranslator() {
+		// TODO Auto-generated method stub
+		return null;
+	}
+	
+	@Override
+	public DiagnosticReport get(String uuid) {
+		// TODO Auto-generated method stub
+		return super.get(uuid);
+	}
+	
+	@Override
+	public DiagnosticReport create(DiagnosticReport newResource) {
+		// TODO Auto-generated method stub
+		return super.create(newResource);
+	}
+	
+	@Override
+	public DiagnosticReport update(String uuid, DiagnosticReport updatedResource) {
+		// TODO Auto-generated method stub
+		return super.update(uuid, updatedResource);
+	}
```

Comment:
See above. You shouldn't need to implement this directly. For the `getDao()` and `getTranslator()` implementations, just add a `@Getter(AccessLevel.PROTECTED)` annotation to the class. The other methods are already defined by the BaseFhirService class and don't need ot be redefined here.